payments-core/src/main/java/com/stripe/android/model/PaymentMethod.kt

```diff
@@ -112,29 +112,30 @@ data class PaymentMethod internal constructor(
     @Parcelize
     enum class Type constructor(
         @JvmField val code: String,
-        @JvmField val isReusable: Boolean
+        @JvmField val isReusable: Boolean,
+        @JvmField val isVoucher: Boolean
```

Comment:
Can this default to false then you only need to set to true for Oxxo?

Reply:
thought about it, but I wanted to make it explicit, so when adding a new type you specifically need to state if it is a voucher or not.